test/test-262-whitelist.txt

```diff
@@ -17518,4 +17514,12 @@ test/language/statements/class/async-gen-method-static/forbidden-ext/b2/cls-decl
 test/language/statements/class/async-gen-method-static/forbidden-ext/b2/cls-decl-async-gen-meth-static-forbidden-ext-indirect-access-prop-caller.js(default)
 test/language/statements/class/async-gen-method-static/forbidden-ext/b2/cls-decl-async-gen-meth-static-forbidden-ext-indirect-access-own-prop-caller-value.js(default)
 test/language/statements/class/elements/syntax/valid/grammar-special-prototype-async-gen-meth-valid.js(default)
-test/language/statements/class/elements/syntax/valid/grammar-special-prototype-async-gen-meth-valid.js(strict mode)
\ No newline at end of file
+test/language/statements/class/elements/syntax/valid/grammar-special-prototype-async-gen-meth-valid.js(strict mode)
```

Comment:
these tests are broken.

In general, `super()` is not allowed in non derived class, but `super.xxx` is allowed.

Reply:
OK, I will check that.

Reply:
I run the test in the local env, it can pass.

Reply:
Then you can remove these newly added tests from this whitelist file.

Reply:
done.